charts/prometheus-postgresql-alerts/values.yaml

```
@@ -0,0 +1,75 @@
+global:
+  additionalRuleLabels: []
+
+defaultRunbookUrl: https://qonto.github.io/database-monitoring-framework/tree/{{chartVersion}}/runbooks/postgresql/{{alertName}}
+rulesGroupName: postgresql.rules
+
+rules:
+
+  PostgreSQLExporterDown:
+    expr: up{} * on (instance) postgres_exporter_build_info{} < 1
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      summary: "{{ $labels.instance }} exporter is down"
+
+  PostgreSQLExporterScraping:
+    expr: avg_over_time(pg_exporter_last_scrape_duration_seconds{}[10m]) > 30
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: "{{ $labels.instance }} scraping take long time"
+
+  PostgreSQLMaxConnections:
+    expr: sum(pg_stat_connections_count) by (container) * 100 / on (container) pg_settings_max_connections{} > 80
+    for: 10m
+    labels:
+      severity: warning
+    annotations:
+      summary: "{{ $labels.container }} is close from the maximum database connections"
+      description: "{{ $labels.container }} uses {{ $value }}% of database connections"
+
+  PostgreSQLReplicationSlotStorage:
```

```suggestion
  PostgreSQLReplicationSlotStorageCloseToMax:
```

Let's try to add the actual error in addition to the component being monitored, so it will be easier to add more alerts for a given component

---

As discussed, I named it `PostgreSQLReplicationSlotStorageLimit`